Realm/RealmSwift/Realm.swift

```diff
@@ -83,6 +83,16 @@ public class Realm {
         }
     }
 
+    // MARK: Writing a Copy
+
+    public func writeCopyToPath(path: String, error: NSErrorPointer) {
+        rlmRealm.writeCopyToPath(path, error: error)
+    }
+
+    public func writeCopyToPath(path: String, encryptionKey: NSData, error: NSErrorPointer) {
```

Comment:
you should leverage Swift's power and combine `writeCopyToPath(path: String, encryptionKey: NSData, error: NSErrorPointer)` and `writeCopyToPath(path: String, error: NSErrorPointer)` into `writeCopyToPath(path: String, error: NSErrorPointer = nil, encryptionKey: NSData? = nil)`.

This will mean we only have a single copy function declared and users can call as `writeCopyToPath("path.realm")` or `writeCopyToPath("path.realm", error: &error)` or `writeCopyToPath("path.realm", error: &error, encryptionKey: key)`.